ui/src/main/resources/edu/wpi/grip/ui/MainWindow.fxml

```diff
@@ -94,6 +94,23 @@
                     </accelerator>
                 </MenuItem>
             </Menu>
+            <Menu text="Tools">
+                <MenuItem text="Settings" onAction="#showProjectSettingsEditor">
+                    <graphic>
+                        <ImageView styleClass="menu-graphic">
+                            <fitWidth>
+                                <DPIUtility fx:constant="SMALL_ICON_SIZE"/>
+                            </fitWidth>
+                            <fitHeight>
+                                <DPIUtility fx:constant="SMALL_ICON_SIZE"/>
+                            </fitHeight>
+                            <image>
+                                <Image url="@icons/settings.png"/>
```

Comment:
You can't use Font Awesome in FXML without the font size getting messed up (Glyph objects don't seem to respect the current font size and when I tried setting it from FXML it didn't work)